spec/ibandit/iban_spec.rb

```diff
@@ -601,6 +601,138 @@
           to eq(account_number: "is the wrong length (should be 10 characters)")
       end
     end
+
+    context "when the IBAN was created with local details for US" do
+      let(:arg) do
+        {
+          country_code: "US",
+          bank_code: bank_code,
+          account_number: account_number,
+        }
+      end
+
+      context "and a 9 digit bank code" do
+        let(:bank_code) { "900000003" }
```

Comment:
Could we use a slightly more complicated routing number? The fact that it is mostly zeros doesn't really test the checksum logic